pkgs/development/python-modules/pytest-regressions/default.nix

```diff
@@ -0,0 +1,48 @@
+{ lib
+, buildPythonPackage
+, fetchpatch
+, fetchPypi
+, pytestCheckHook
+, matplotlib
+, numpy
+, pandas
+, pillow
+, pytest
+, pytest-datadir
+, pyyaml
+, setuptools-scm
+}:
+
+buildPythonPackage rec {
+  pname = "pytest-regressions";
+  version = "2.2.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "15a71f77cb266dd4ca94331abe4c339ad056b2b2175e47442711c98cf6d65716";
+  };
+  patches = [
+    # Make pytest-regressions compatible with NumPy 1.20.
```

Comment:
It helps other maintainers if the there is a statement that the change will be part of the next release.